modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
 ----
 SHOW INDEXES YIELD *
 ----
-====
+
+======
+[.include-with-GQLSTATUS-code]
+======
+Query::
++
+[source,cypher]
+----
+CYPHER runtime=pipelined SHOW INDEXES YIELD *
+----
+
+Returned GQLSTATUS code::
+01N40
+
+Returned Status Description::
+warn: runtime unsupported.
+The query cannot be executed with `runtime=pipelined` is used.
```

Comment:
```suggestion
The query cannot be executed with `runtime=pipelined`, `runtime=slotted` is used.
```

Reply:
We can also add the other example:

Parallel runtime does not support write queries
`EXPLAIN CYPHER runtime=parallel CREATE (n)`
The returned GQLSTATUS code will be the same.

I wonder if we can rename the example headings to remove the "An example of" part.